src/fontra/views/editor/panel-transformation.js

```diff
@@ -101,6 +102,46 @@ export default class TransformationPanel extends Panel {
       skewY: 0,
       customDistributionSpacing: null,
     };
+    this.registerActions();
+  }
+
+  registerActions() {
+    const topic = "0070-action-topics.selection-transformations";
+
+    const registerActions = [
+      ["align.left", alignLeft],
+      ["align.center", alignCenter],
+      ["align.right", alignRight],
+      ["align.top", alignTop],
+      ["align.middle", alignMiddle],
+      ["align.bottom", alignBottom],
+      ["distribute.horizontally", distributeHorizontally],
+      ["distribute.vertically", distributeVertically],
+    ];
+    for (const [keyPart, moveDescriptor] of registerActions) {
+      registerAction(
+        `action.selection-transformation.${keyPart}`,
+        { topic, titleKey: `sidebar.selection-transformation.${keyPart}` },
+        () => this.moveObjects(moveDescriptor)
+      );
+    }
+
+    const registerActionsPathOperations = [
```

Comment:
And for this one I'd prefer `pathActions`.

Reply:
done.